crates/matrix-sdk/src/room/timeline/mod.rs

```diff
@@ -69,6 +74,7 @@ pub struct Timeline {
     _fully_read_handler_guard: Option<EventHandlerDropGuard>,
     #[cfg(feature = "e2e-encryption")]
     _room_key_handler_guard: EventHandlerDropGuard,
+    _forwarded_room_key_handler_guard: EventHandlerDropGuard,
```

Comment:
Why an `_` prefix?

Reply:
Because it's intentionally never used outside of the implicit `Drop` impl and otherwise triggers "field is never read" warnings. I actually want to refactor this a bit though, in the next PR.